benchmarks/domains/synthetic_2C1D_1C.py

```diff
@@ -60,24 +62,26 @@ def synthetic_2C1D_1C(settings: ConvergenceExperimentSettings) -> DataFrame:
         NumericalDiscreteParameter("z", (1, 2, 3, 4)),
     ]
 
-    objective = NumericalTarget(name="target", mode=TargetMode.MAX).to_objective()
-    search_space = SearchSpace.from_product(parameters=parameters)
+    target = NumericalTarget(name="target", mode="MAX")
+    searchspace = SearchSpace.from_product(parameters=parameters)
 
     scenarios: dict[str, Campaign] = {
         "Random Recommender": Campaign(
-            searchspace=search_space,
+            searchspace=searchspace,
             recommender=RandomRecommender(),
-            objective=objective,
+            objective=target,
         ),
         "Default Recommender": Campaign(
-            searchspace=search_space,
-            objective=objective,
+            searchspace=searchspace,
+            objective=target,
```

Comment:
even thoughw e ahve implicit conversion I dont think its best practice to use it in code that others might take as blueprints

Reply:
alright, changed it